pages/advanced-algorithms/available-algorithms/migrate.mdx

```diff
@@ -27,17 +27,65 @@ A module that contains procedures describing graphs on a meta-level.
 
 ## Procedures
 
+### `db2()`
+
+With the `migrate.db2()` procedure you can access IBM DB2 and migrate your data
+to Memgraph. The result table is converted into a stream, and the returned rows can
+be used to create graph structures. The value of the `config` parameter must be
+at least an empty map. If `config_path` is passed, every key-value pair from
+JSON file will overwrite any values in `config` file.
+
+{<h4> Input: </h4>}
+
+* `table_or_sql: str` ➡ Table name or an SQL query. When the table name is specified, the module
+        will migrate all the rows from the table. In the case that a SQL query is provided, the module
+        will migrate the rows returned from the queries.
+* `config: mgp.Map` ➡ Connection configuration parameters (as in `ibm_db.connect`).
+* `config_path` ➡ Path to the JSON file containing configuration parameters (as in `ibm_db.connect`).
+* `params: mgp.Nullable[mgp.Any] (default=None)` ➡ Optionally, queries can be parameterized. In that case, `params` provides parameter values.
+    
+{<h4> Output: </h4>}
+
+* `row: mgp.Map`: The result table as a stream of rows.
+
+{<h4> Usage: </h4>}
+
+To get all data from database in form of map, use the following query:
+
+```cypher
+CALL migrate.db2('example_table', {user:'memgraph',
+        password:'password',
+        host:'localhost',
+        database:'demo_db'} )
+YIELD row
+RETURN row;
+```
+
+In the case you want to migrate specific results from a SQL query, it is enough to modify the 
+first argument of the query module call:
+
+```cypher
+CALL migrate.db2('SELECT * FROM example_table', {user:'memgraph',
+        password:'password',
+        host:'localhost',
+        database:'demo_db'} )
+YIELD row
+RETURN count(row);
```

Comment:
Why `count(row)` here?

Reply:
modified the examples to be without counts and with arbitrary cypher functionalities.